src/browser_app_skeleton/package.json

```diff
@@ -8,18 +8,18 @@
   },
   "scripts": {
     "heroku-postbuild": "yarn prod",
-    "dev": "yarn run webpack --progress --hide-modules --color --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "watch": "yarn run webpack --watch --hide-modules --color --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "prod": "NODE_ENV=production yarn run webpack --progress --hide-modules --color --config=node_modules/laravel-mix/setup/webpack.config.js"
+    "dev": "mix",
```

Comment:
When you run `yarn` on a new lucky project, will you get a global `mix` executable? Or does this need to be `yarn run mix` so it scopes to this project?

Reply:
`yarn install` was sufficient for me. I think `package.json` scripts have `node_modules/bin` in their path.